src/router/index.js

```diff
@@ -38,10 +35,10 @@ const routes = [
   },
 ];
 
-const router = new VueRouter({
-  mode: 'history',
+const router = createRouter({
+  history: createWebHistory(process.env.BASE_URL),
   base: process.env.BASE_URL,
```

Comment:
I guess the base prop can now also be removed?
https://next.router.vuejs.org/guide/migration/#moved-the-base-option